CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+# Contributing Guidelines
+
+Thanks for your interest in contributing to `Thoth Station`.
+
+## Reporting Bugs
+
+Before creating bug reports, please check a [list of known issues](https://github.com/issues?q=is%3Aopen+is%3Aissue+archived%3Afalse+user%3Athoth-station+sort%3Acreated-asc) to see
+if the problem has already been reported (or fixed in a master branch).
+
+**Note:** If you find a **Closed** issue that seems like it is the same thing that you're experiencing, open a new issue and include a link to the original issue in the body of your new one.
+You can also comment on the closed issue to indicate that upstream should provide a new release with a fix.
+
+### Suggesting Enhancements
+
+Enhancement suggestions are tracked as GitHub issues.
+When you are creating an enhancement issue, **use a clear and descriptive title** and **provide a clear description of the suggested enhancement** in as many details as possible.
+
+## Guidelines for Developers
+
+If you would like to contribute code to the `Thoth` project, this section is for you!
+
+### Is this your first contribution?
+
+Please take a few minutes to read GitHub's guide on [How to Contribute to Open Source](https://opensource.guide/how-to-contribute/).
+It's a quick read, and it's a great way to introduce yourself to how things work behind the scenes in open-source projects.
+
+### Dependencies
+
+If you are introducing a new dependency, please make sure it's added to:
+
+- `Pipfile`
+- `setup.py` if its a package
```

Comment:
don't we place it in`requirements.txt` and read it from setup.py?

Reply:
@fridex what is the good practice that we want here?

Reply:
We use `requirements.txt` files for library releases (requirements on direct dependencies) and Pipfile/Pipfile.lock for components (fully pinned down stack):

- Pipfile/Pipfile.lock if the repo is a component built by Python s2i
- `requirements.txt` (read from `setup.py`) if the repo is a library released to PyPI

The two can be used at the same time.

Reply:
BTW we could introduce a check to make sure they are in sync (requirements.txt corresponds to Pipfile):

Related: https://github.com/thoth-station/kebechet/issues/705